sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3056,7 +3059,6 @@ class Dataset[T] private[sql](
    * view, e.g. `SELECT * FROM global_temp.view1`.
    *
    * @throws AnalysisException if the view name is invalid or already exists
-   *
```

Comment:
why all these comment changes? could you revert them

Reply:
Sorry for this, I'll revert the IDE changes.